tests/Benchmarks/Microsoft.Experimental.Collections/MultiValueDictionaryPerformanceTests.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Xunit;
+using BenchmarkDotNet.Attributes;
+
+namespace Microsoft.Collections.Extensions.Tests
+{
+    public class MultiValueDictionaryPerformanceTests
+    {
+        [Params(1000, 10000, 10000)]
+        public int Size { get; set; }
+
+        public int RandomKey { get; set; }
+
+        public MultiValueDictionary<int, int> dict;
+        
+        [GlobalSetup(Targets = new[] { nameof(Add), nameof(Clear), nameof(GetItem), nameof(GetKeys), nameof(ContainsValue) })]
+        public void CreateMultiValueDictionary_SingleValues()
+        {
+            dict = new MultiValueDictionary<int, int>();
+            Random rand = new Random(11231992);
+
+            while (dict.Count < Size)
+                dict.Add(rand.Next(), rand.Next());
+        }
+
+        [GlobalSetup(Targets = new[] { nameof(TryGetValue), nameof(ContainsKey) })]
+        public void CreateMultiValueDictionary_AddRandomKey()
+        {
+            CreateMultiValueDictionary_SingleValues();
+            RandomKey = new Random(837322).Next(0, 400000);
+            dict.Add(RandomKey, 12);
+        }
+
+        [Benchmark]
+        public void Add()
+        {
+            for (int i = 0; i <= 20000; i++)
+            {
+                dict.Add(i * 10 + 1, 0); dict.Add(i * 10 + 2, 0); dict.Add(i * 10 + 3, 0);
+                dict.Add(i * 10 + 4, 0); dict.Add(i * 10 + 5, 0); dict.Add(i * 10 + 6, 0);
+                dict.Add(i * 10 + 7, 0); dict.Add(i * 10 + 8, 0); dict.Add(i * 10 + 9, 0);
+            }
+        }
+
+        public List<int> values;
```

Comment:
nit: Should this be a private field? If not, use PascalCasing.

Reply:
Will change to be private.